jib-plugins-common/src/main/java/com/google/cloud/tools/jib/plugins/common/PluginConfigurationProcessor.java

```diff
@@ -289,16 +273,34 @@ static PluginConfigurationProcessor processCommonConfiguration(
    * @param rawConfiguration raw configuration data
    * @param projectProperties used for providing additional information
    * @return the base image
+   * @throws IncompatibleBaseImageJavaVersionException
```

Comment:
Missing description (although doesn't matter that much since it's package private).

Reply:
Will add. The exception class is not package-private, so makes sense to add it.